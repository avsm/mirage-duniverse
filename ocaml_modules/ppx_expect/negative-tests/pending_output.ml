module Expect_test_config : Expect_test_config.S with type 'a IO.t = 'a = struct
  include Expect_test_config

  let flushed () = print_string "blah\n"; false
end

let%expect_test _ =
  print_endline "foo";
  [%expect {| foo
           |}]
;;

