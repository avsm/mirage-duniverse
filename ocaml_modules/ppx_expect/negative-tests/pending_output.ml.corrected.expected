module Expect_test_config : Expect_test_config.S with type 'a IO.t = 'a = struct
  include Expect_test_config

  let flushed () = print_string "blah\n"; false
end

let%expect_test _ =
  print_endline "foo";
  [%expect {| foo
           |}];
  [%expect {|
    blah
    blah
    blah
    blah
    blah
    blah
    blah
    blah
    blah
    blah

    STOPPED COLLECTING OUTPUT AFTER 10 FLUSHING ATTEMPS
    THERE MUST BE A BACKGROUND JOB PRINTING TO STDOUT |}]
;;

